build.yaml

```diff
@@ -931,7 +931,7 @@ steps:
 
      python3 -m flake8 --config setup.cfg $SITE_PACKAGES/hail $SITE_PACKAGES/hailtop || exit_status=$?
      python3 -m pylint --rcfile pylintrc hailtop || exit_status=$?
-     (cd $SITE_PACKAGES/hailtop && mypy batch google_storage.py) || exit_status=$?
+     (cd $SITE_PACKAGES/hailtop/ && mypy --config-file /setup.cfg utils batch google_storage.py) || exit_status=$?
```

Comment:
Does this one need `--ignore-missing-imports`?

Reply:
Oops.  No, I removed it from the other step.  We shouldn't specify the option on teh command line because we're setting it in the setup.cfg now.

Reply:
Did you forget a commit? It looks like one step still has the global flag.

Reply:
Fixed.